.github/workflows/reuse.yml

```diff
@@ -0,0 +1,254 @@
+name: If Levaithan was reusuable?
+
+on:
+  workflow_call:
+    inputs:
+      LEVIATHAN_REF:  # id of input
+        description: "Leviathan branch, or tag, or commit to test with. Examples: 'refs/heads/<branchName>', 'refs/tags/<tagName>', '<commitId>'"
+        required: false
+        type: string
+        default: 'refs/heads/master'
+      # CONFIG:
+      #   description: 'path to config.js'
+      #   required: false
+      #   default: './suites/config.js'
+      SUITES:
+        description: 'path to test folders'
+        required: false
+        type: string
+        default: './suites'
+      # TEST_SUITE_NAME:
+      #   description: 'which test suite to run'
+      #   required: false
+      #   type: string
+      WORKSPACE:
+        description: 'path to WORKSPACE'
+        required: false
+        type: string
+        default: './.leviathan-inputs'
+      DEVICE_TYPE:
+        description: "slug of device type"
```

Comment:
this is the slug from contracts, not the yocto machine name, correct?

Reply:
Yeah, this needs some discussion of what it will be. The good thing is that they are under user control to provide the right values.